openmmapi/include/openmm/NonbondedForce.h

```diff
@@ -208,6 +213,16 @@ class OPENMM_EXPORT NonbondedForce : public Force {
      */
     void getPMEParameters(double& alpha, int& nx, int& ny, int& nz) const;
     /**
+     * Get the parameters to use for dispersion term in LJ-PME calculations.  If alpha is 0 (the default),
+     * these parameters are ignored and instead their values are chosen based on the Ewald error tolerance.
+     *
+     * @param[out] dalpha   the separation parameter
+     * @param[out] dnx      the number of dispersion grid points along the X axis
+     * @param[out] dny      the number of dispersion grid points along the Y axis
+     * @param[out] dnz      the number of dispersion grid points along the Z axis
+     */
+    void getLJPMEParameters(double& dalpha, int& dnx, int& dny, int& dnz) const;
```

Comment:
Sure, that's fine with me.  My intention was to make it absolutely clear that this only sets the dispersion PME parameters, but it is probably cleaner without the prefixes.  I'll remove them during my next set of commits.